src/metabase/automagic_dashboards/populate.clj

```diff
@@ -43,9 +44,22 @@
         :location "/")
       (create-collection! "Automatically Generated Dashboards" "#509EE3" nil nil)))
 
-(def colors
+(defn- get-whitelabeled-colors
+  "Get color from application-colors or default color."
+  [default-colors]
+  (let [whitelabeled-color-names [:brand :accent1 :accent2 :accent3 :accent4 :accent5 :accent6 :accent7]]
+    (->> default-colors
+         (map vector whitelabeled-color-names)
+         (map (fn [[whitelabeled-color-name default-color]]
+                (whitelabeled-color-name (public-settings/application-colors) default-color)))
+         (into []))))
+
+(defn colors
   "Colors used for coloring charts and collections."
-  ["#509EE3" "#9CC177" "#A989C5" "#EF8C8C" "#f9d45c" "#F1B556" "#A6E7F3" "#7172AD"])
+  []
+  (get-whitelabeled-colors ["#509EE3" "#88BF4D" "#A989C5" "#EF8C8C" "#F9D45C" "#F2A86F" "#98D9D9" "#7172AD"]))
```

Comment:
Change some colors to match `colors.ts`, they were close but weren't the same color.
https://github.com/metabase/metabase/blob/8b341b1ac3b1c95d891058966bfb30b264b9285f/frontend/src/metabase/lib/colors.ts#L10-L18

From eye-balling I get that the colors were defined like this
`[brand accent1 accent2 accent3 accent4 accent5 accent6 accent7]`